spec/features/categories/user_edits_an_existing_category_spec.rb

```diff
@@ -0,0 +1,19 @@
+require 'rails_helper'
+
+describe "User can edit existing category" do
+  scenario "they can go to edit page and put new title" do
```

Comment:
what happens if a user doesn't enter valid data while trying to edit? we should have a test to make sure that a user sees what we expect them to see 👍